
INIT: Using found mnist data
                                                     img  ...   test
0      [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
1      [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
2      [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
3      [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
6      [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
...                                                  ...  ...    ...
69995  [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
69996  [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
69997  [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
69998  [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
69999  [  0   0   0   0   0   0   0   0   0   0   0  ...  ...  False
[55994 rows x 4 columns]
2022-03-31 14:03:39.103161: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: UNKNOWN ERROR (34)
2022-03-31 14:03:39.103264: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (b371895a6098): /proc/driver/nvidia/version does not exist
2022-03-31 14:03:39.103771: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-03-31 14:03:39.105788: W tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 351194368 exceeds 10% of free system memory.
2022-03-31 14:04:57.157069: W tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 175597184 exceeds 10% of free system memory.
INIT: Finished creating train dataset
INIT: Finished creating test dataset
Setup Complete, Starting training:
2022-03-31 14:05:08.438682: W tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 87845632 exceeds 10% of free system memory.
[2 0 4 8 0 3 1 8 0 7 9 4 4 1 0 7 8 1 2 7 1 6 4 7 3 3 5 3 4 6 6 3]
2022-03-31 14:06:14.892356: W tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 351194368 exceeds 10% of free system memory.
2022-03-31 14:07:20.636727: W tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 351194368 exceeds 10% of free system memory.
Traceback (most recent call last):
  File "RunTest.py", line 172, in <module>
    main(args)
  File "RunTest.py", line 120, in main
    d += len(batch[1].numpy()) #TODO update so we can see the images used per class
TypeError: 'int' object is not iterable